### DISCUSSION TITLE
Nice to have things

### DISCUSSION THREAD

**Discussion by MaxDKCrab** [InaSLew/AutumnGameJam-PumpkinMadness#11] [Ideas]
What do we want to implement when we have all the must haves?

**Comment by SearWaizy** [comment]
**Destructible platforms**
Only the player will be able to destroy platforms. If the player stands on a set platform for X amount of time the platform destroys and respawns after 2 sec.

* The grass platforms **are not** destructible.
* The hay platforms will destory if the player stands on it more than **3 sec**.
* The vine platform will destroy if the player stands on it more than **8 sec**.

**Comment by MaxDKCrab** [comment]
Tutorial Section!
Essentially just some ui element showing the controls to the player
dissapears after a few seconds
if we have time make it skippable

**Comment by MaxDKCrab** [comment]
Large ground enemy type?

Pickups?

Enemies dropping "health globes" 

if pickups and/or health globes implemented, having enemies with drops indicate to the player that they do so

Boss fight!?

New level elements as time goes on?
